packages/react-dom/src/__tests__/ReactErrorBoundaries-test.internal.js

```diff
@@ -644,6 +644,39 @@ describe('ReactErrorBoundaries', () => {
     expect(container3.firstChild).toBe(null);
   });
 
+  it('logs a single error when using error boundary', () => {
+    const container = document.createElement('div');
+    expect(() =>
+      ReactDOM.render(
+        <ErrorBoundary>
+          <BrokenRender />
+        </ErrorBoundary>,
+        container,
+      ),
+    ).toWarnDev('The above error occurred in the <BrokenRender> component:', {
+      logAllErrors: true,
```

Comment:
Added a new option for `toWarnDev` which will log all errors

Without this, `toWarnDev` skips "The above error" errors [here](https://github.com/facebook/react/blob/b2adcfba32d133aabdbc182bba1001e729060abe/scripts/jest/shouldIgnoreConsoleError.js#L11)